heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -111,11 +112,24 @@ public static Process runASyncProcess(
       boolean verbose, String[] command, File workingDirectory) {
     // Log the command for debugging
     LOG.log(Level.FINE, "$> {0}", Arrays.toString(command));
+    // Log the working directory, this will make people fast locate the
+    // directory to find the log files
+    LOG.log(Level.INFO, "working dir: {0}", workingDirectory.getAbsolutePath());
```

Comment:
We have already logged the location of working directory when a topology is submitted successfully via LocalScheduler: https://github.com/twitter/heron/blob/master/heron/schedulers/src/java/com/twitter/heron/scheduler/local/LocalLauncher.java#L83

Also, working directory is just a concept existing only in LocalScheduler, while ShellUtils is a common utils method used in a lot of places.

A better alternative can be log the working directory no matter a topology is submitted successfully or not at the start for LocalScheduler.